src/AppRoot.tsx

```diff
@@ -221,33 +256,47 @@ class AppRoot extends React.Component<RouteComponentProps, State> {
     checkCardInterval = 0 // To indicate setInterval is not running.
   }
 
-  public markVoterCardUsed = async (
-    { ballotPrinted } = { ballotPrinted: true }
-  ) => {
+  public togglePauseProcessingUntilNoCardPresent = (b?: boolean) => {
```

Comment:
What is `b` here? It seems like passing `true` to this method will force `pauseProcessingUntilNoCardPresent` to be `true`, but passing `false` (or nothing) will toggle the value. This feels a bit confusing. Should these be separate methods? One that toggles and one that just sets it?

Reply:
Ah, I see my failure in logic. I could "toggle" only on `undefined`, but since I'm only calling method with a value, I'll remove ability to toggle.